picard/webservice/__init__.py

```diff
@@ -425,7 +427,8 @@ def _handle_reply(self, reply, request):
                     self._handle_redirect(reply, request, redirect)
                 elif request.response_parser:
                     try:
-                        document = request.response_parser(reply)
+                        WebService.last_raw_reply = reply.readAll()
```

Comment:
Really unsure about this.

Reply:
Agreed, I had documented it and explained it best I could.  not much other options without drastic code changes.  Its a moot point now, as phw has asked to remove the display of json, there is no need to capture it anymore.  I'll be removing this code.